script/Arcueid/Day 8/08_00_ARC08_2B.txt

```diff
@@ -112,7 +112,7 @@ He's right.
 -- ここは遠野邸前の国道で、通るのは乗用車ぐらい。
 // 国道 is decidedly not a western conception of 'highway'
 We're on the road in front of the Tohno residence,
- and the only traffic it usually gets is the odd car.
+ and the only traffic it usually gets is the odd car. //What does it mean by Odd car?
```

Comment:
It means that there is not much traffic, only cars pass every so often. Its a standard phrase.